paddle/framework/operator.h

```diff
@@ -110,29 +98,32 @@ class OperatorBase {
   std::shared_ptr<std::unordered_map<std::string, int>> in_out_idxs_;
 };
 
-class KernelContext {
+class OperatorContext {
  public:
-  KernelContext(const OperatorBase* op, const std::shared_ptr<Scope>& scope,
-                const platform::DeviceContext& device_context)
-      : op_(*op), scope_(scope), device_context_(device_context) {}
+  OperatorContext(const OperatorBase* op, const std::shared_ptr<Scope>& scope)
+      : op_(*op), scope_(scope) {}
```

Comment:
we donot need to check because context will only be construct inside a op, so op will nevel be null. And so be need not to use std::shared_ptr<OperatorBase>